DFC.ServiceTaxonomy.Neo4j/Services/GraphCluster.cs

```diff
@@ -31,6 +31,11 @@ public Task<List<T>> Run<T>(string replicaSetName, IQuery<T> query)
             return _graphReplicaSets[replicaSetName].Run(query);
         }
 
+        public Task<List<T>> Run<T>(string replicaSetName, IEnumerable<IQuery<T>> queries)
```

Comment:
Would you mind changing the 2 query methods to use params, in a similar manner to the command Run().